Src/library/wel/windows/wel_window.e

```diff
@@ -2252,12 +2252,16 @@ feature {WEL_WINDOW} -- Windows bug workaround
 			-- This is a limitation that the windows kernel imposes on the messaging system
 			-- so that the callstack is not too deep. To circumvent this we simply call
 			-- `PostMessage' when it fails to resize the window.
+			-- The current approach use the Per-Monitor DPI-aware variant.
 		require
 			exists: exists
 		local
 			l_diff: BOOLEAN
 			l_flags: INTEGER
 			l_pos: WEL_WINDOW_POS
+			l_dpi: INTEGER
```

Comment:
I don't think this is the place where change in size computation should occur. It should just be in the font size computation.